spec/player_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "./lib/player"
+
+RSpec.describe Player do
+  it 'Tests the player name' do
+    player = Player.new('player', 'T')
+    expect(player.name).to eql('player')
+  end
+
+  it 'Tests the player symbols' do
+    player = Player.new('player', 'T')
```

Comment:
You can use repeating this you can write **let(player) {Player.new('player', 'T'}**

Reply:
Thanks!!